### PR TITLE
replace count() usage to empty() when possible

### PR DESCRIPTION
**Pull request by samsonasik**
ref: http://maettig.com/code/php/php-performance-benchmarks.php#empty

### CODE REVIEW

File: src/Tool/ConfigDumperCommand.php [hunk 1]
```diff
@@ -124,7 +124,7 @@ public function __invoke(array $args)
      */
     private function parseArgs(array $args)
     {
-        if (! count($args)) {
+        if (empty($args)) {
```
Comment (Ocramius) [inline src/Tool/ConfigDumperCommand.php]: Can be replaced with `if ($args) {`

File: src/Tool/ConfigDumperCommand.php [hunk 2]
```diff
@@ -140,7 +140,7 @@ private function parseArgs(array $args)
             $arg1 = array_shift($args);
         }
 
-        if (! count($args)) {
+        if (empty($args)) {
```
Comment (Ocramius) [inline src/Tool/ConfigDumperCommand.php]: Can be replaced with `if ($args) {`

File: src/Tool/FactoryCreatorCommand.php [hunk 3]
```diff
@@ -104,7 +104,7 @@ public function __invoke(array $args)
      */
     private function parseArgs(array $args)
     {
-        if (! count($args)) {
+        if (empty($args)) {
```
Comment (Ocramius) [inline src/Tool/FactoryCreatorCommand.php]: Can be replaced with `if ($args) {`

**Comment by samsonasik** [comment]
@Ocramius I've updated `empty($args)` with `! $args`

**Comment by Ocramius** [comment]
LGTM :+1:

**Pull request merged**